packages/react-components/react-search/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -1,17 +1,25 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { Input, InputState } from '@fluentui/react-input';
 
 export type SearchBoxSlots = {
-  root: Slot<'div'>;
+  // Root of the component, wrapping the inputs
+  root: NonNullable<Slot<typeof Input>>;
+
+  // Last element in the input, within the input border
+  dismiss?: Slot<'span'>;
+
+  // Element after the input text, within the input border
+  contentAfter?: Slot<'span'>;
 };
 
 /**
  * SearchBox Props
  */
-export type SearchBoxProps = ComponentProps<SearchBoxSlots> & {};
+export type SearchBoxProps = ComponentProps<SearchBoxSlots>;
 
 /**
  * State used in rendering SearchBox
  */
-export type SearchBoxState = ComponentState<SearchBoxSlots>;
-// TODO: Remove semicolon from previous line, uncomment next line, and provide union of props to pick from SearchBoxProps.
-// & Required<Pick<SearchBoxProps, 'propName'>>
+export type SearchBoxState = ComponentState<SearchBoxSlots> &
+  Pick<InputState, 'size'> &
+  Required<Pick<SearchBoxProps, 'disabled'>>;
```

Comment:
Since you provide a default value on the useSearchBox hook, you should make size required:

```suggestion
export type SearchBoxState = ComponentState<SearchBoxSlots> &
  Required<Pick<InputState, 'size'>> &
  Required<Pick<SearchBoxProps, 'disabled'>>;
```

Reply:
I believe it's already a required prop on InputState, though thinking about this more, it might be more readable to just combine them as:
`Required<Pick<SearchBoxProps, 'disabled' | 'size'>>`